hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/ethereum/EthereumTransitionLogic.java

```diff
@@ -120,13 +125,20 @@ public void doStateTransition() {
 
         // Revoke the relayer's key for Ethereum operations
         txnCtx.swirldsTxnAccessor().getSigMeta().revokeCryptoSigsFrom(txnCtx.activePayerKey());
-        if (synthTxn.hasContractCall()) {
-            delegateToCallTransition(callerNum.toId(), synthTxn, relayerId, maxGasAllowance, userOfferedGasPrice);
-        } else {
-            delegateToCreateTransition(callerNum.toId(), synthTxn, relayerId, maxGasAllowance, userOfferedGasPrice);
+        try {
+            if (synthTxn.hasContractCall()) {
+                delegateToCallTransition(callerNum.toId(), synthTxn, relayerId, maxGasAllowance, userOfferedGasPrice);
+            } else {
+                delegateToCreateTransition(callerNum.toId(), synthTxn, relayerId, maxGasAllowance, userOfferedGasPrice);
+            }
+        } catch (InvalidTransactionException e) {
```

Comment:
Is `InvalidTransactionException` the only one that should be caught here?

Reply:
hmm, my thinking is we want to catch all _validation_ exceptions (thrown for example [here](https://github.com/hashgraph/hedera-services/blob/06b8d8ae123ece1ab72c35f43c1097d1dfb0dc9b/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogic.java#L147)) and populate the record stream with the message they contain. I'm not sure we would want to catch _any_ exception here and possibly include information in the record stream that's more sensitive. 
@tinker-michaelj wdyt?

Reply:
My only concern is the bugs/system error part.
If i submit an execution I should be able to get a hint of why it failed and the result details regardless.
We want to avoid submitting a transaction and not knowing details around gas consumed and result details.
If this PR doesn't address bugs/system errors we've definitely improved the issue but haven't resolved it.
I'm fine with a follow up in 0.45 if necessary but I'm not certain we should leave as is in the improved state.
@stoqnkpL if we don't cover the rest please open a ticket to describe what's left. It's possible we triage and say won't fix but let's still capture it